src/main/java/io/phdata/streamliner/schemadefiner/util/StreamlinerUtil.java

```diff
@@ -411,4 +409,11 @@ public static void createFile(String... files) {
               }
             });
   }
+
+  public static final Set<String> snowflakeStringDataType =
+      new HashSet<>(
+          Arrays.asList(
+              new String[] {
+                "VARCHAR", "CHAR", "CHARACTER", "STRING", "TEXT", "BINARY", "VARBINARY"
+              }));
```

Comment:
Since this is Java 1.8, we have to do it like this. Or else I think we would have to create a method where we will use Set add() method to add all the values and then return the Set.

Reply:
No problem, looks fine